pkg/networkservice/common/connect/server_test.go

```diff
@@ -326,9 +328,9 @@ func TestConnectServer_RequestParallel(t *testing.T) {
 	wg.Wait()
 	wg.Add(parallelCount)
 
-	require.Equal(t, int32(parallelCount), serverClient.count)
-	require.Equal(t, int32(parallelCount), serverA.count)
-	require.Equal(t, int32(parallelCount), serverNext.count)
+	assert.Equal(t, int32(parallelCount), serverClient.count)
+	assert.Equal(t, int32(parallelCount), serverA.count)
+	assert.Equal(t, int32(parallelCount), serverNext.count)
```

Comment:
Why we change it here and don't change below?

Reply:
If test doesn't reach `barrier.Done()` (line 335) it breaks, so we can't use here `require` and need `assert`.